org.elysium.parent/org.elysium.ui/src/org/elysium/ui/compiler/CompilerJob.java

```diff
@@ -206,6 +206,8 @@ private void updateSyntax(IProgressMonitor monitor, CompilerConsole console) {
 		try {
 			ProcessBuilder processBuilder = SyntaxUpdaterProcessBuilderFactory.get(file);
 			prepareProcessBuilder(processBuilder);
+			List<String> command = processBuilder.command();
+			command.add(command.size() - 1, "--edit"); //$NON-NLS-1$
```

Comment:
Please move this to SyntaxUpdaterProcessBuilderFactory because it belongs there (of course without the condition).